Build failure: slade

### Nixpkgs version

- Stable (24.11)

### Steps to reproduce

1. `nix profile install nixpkgs#slade`

### Can Hydra reproduce this build failure?

Please select the Hydra Status.

### Link to Hydra build job

_No response_

### Relevant log output

```console
--- stdout ---
orest ~ > NIXPKGS_ALLOW_INSECURE=1 nix profile install nixpkgs#slade --impure
error: builder for '/nix/store/ws2kinq716dmvfbnbbbp18zffa5kf16y-wxwidgets-3.2.6.drv' failed with exit code 1;
       last 15 log lines:
       > Running phase: unpackPhase
       > unpacking source archive /nix/store/xn88dcb93mwy7lj33k1lg2fljgkfbzcr-source
       > source root is source
       > Running phase: patchPhase
       > applying patch /nix/store/zmh3sbj1mj617lr6q8npzd026skyrs03-425d9455e8307c1267a79d47d77e3dafeb4d86de.patch
       > patching file include/wx/arrstr.h
       > patching file include/wx/defs.h
       > Reversed (or previously applied) patch detected!  Assume -R? [n]
       > Apply anyway? [n]
       > Skipping patch.
       > 1 out of 1 hunk ignored -- saving rejects to file include/wx/defs.h.rej
       > patching file include/wx/dynarray.h
       > patching file interface/wx/arrstr.h
       > patching file interface/wx/dynarray.h
       > patching file tests/arrays/arrays.cpp
       For full logs, run 'nix log /nix/store/ws2kinq716dmvfbnbbbp18zffa5kf16y-wxwidgets-3.2.6.drv'.
error: 1 dependencies of derivation '/nix/store/sliwlbvwzrsdh3qk3fbxb4n3f22f2wkk-slade-3.2.5.drv' failed to build


--- full logs ---
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/xn88dcb93mwy7lj33k1lg2fljgkfbzcr-source
source root is source
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
applying patch /nix/store/zmh3sbj1mj617lr6q8npzd026skyrs03-425d9455e8307c1267a79d47d77e3dafeb4d86de.patch
patching file include/wx/arrstr.h
patching file include/wx/defs.h
Reversed (or previously applied) patch detected!  Assume -R? [n]
Apply anyway? [n]
Skipping patch.
1 out of 1 hunk ignored -- saving rejects to file include/wx/defs.h.rej
patching file include/wx/dynarray.h
patching file interface/wx/arrstr.h
patching file interface/wx/dynarray.h
patching file tests/arrays/arrays.cpp
```

### Additional context

- Hydra is dead at the moment of issue creation:
500 Internal Server Error
DBIx::Class::Storage::DBI::_dbh_execute(): DBI Exception: DBD::Pg::st execute failed: ERROR: canceling statement due to statement timeout [for Statement "SELECT me.id, me.finished, me.timestamp, me.jobset_id, me.job, me.nixname, me.description, me.drvpath, me.system, me.license, me.homepage, me.maintainers, me.maxsilent, me.timeout, me.ischannel, me.iscurrent, me.priority, me.globalpriority, me.starttime, me.stoptime, me.iscachedbuild, me.buildstatus, me.size, me.closuresize, me.releasename, me.keep, me.notificationpendingsince FROM builds me LEFT JOIN buildoutputs buildoutputs ON buildoutputs.build = me.id WHERE ( buildoutputs.path ILIKE ? ) ORDER BY id desc LIMIT ?" with ParamValues: 1='%slade%', 2='10'] at /nix/store/39q79k6mlnsq7b8fc6l5pm5g088nps3j-hydra-0.1.20250112.5385d42/libexec/hydra/lib/Hydra/Controller/Root.pm line 517

### System metadata

orest ~ > nix-shell -p nix-info --run "nix-info -m"
fetching path input 'path:/nix/store/mr3pq6jnd0j7vny01q8r8a2ij8z2mn2d-source'
install: cannot remove '/tmp/env-vars': Operation not permitted
 - system: `"x86_64-linux"`
 - host os: `Linux 6.6.68, NixOS, 24.11 (Vicuna), 24.11.20241231.edf04b7`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Lix, like Nix) 2.91.1
System type: x86_64-linux
Additional system types: i686-linux, x86_64-v1-linux, x86_64-v2-linux, x86_64-v3-linux
Features: gc, signed-caches
System configuration file: /etc/nix/nix.conf
User configuration files: /home/orest/.config/nix/nix.conf:/etc/xdg/nix/nix.conf:/home/orest/.nix-profile/etc/xdg/nix/nix.conf:/nix/profile/etc/xdg/nix/nix.conf:/home/orest/.local/state/nix/profile/etc/xdg/nix/nix.conf:/etc/profiles/per-user/orest/etc/xdg/nix/nix.conf:/nix/var/nix/profiles/default/etc/xdg/nix/nix.conf:/run/current-system/sw/etc/xdg/nix/nix.conf:/nix/store/qvpmfwqfigj1y1jqpdz075ycxgqkhdw5-gnome-settings-daemon-47.2/etc/xdg/nix/nix.conf
Store directory: /nix/store
State directory: /nix/var/nix
Data directory: /nix/store/bimspgs3cpapsbz7a2vi690916fgrvs5-lix-2.91.1/share`
 - channels(root): `""`
 - nixpkgs: `/nix/store/mr3pq6jnd0j7vny01q8r8a2ij8z2mn2d-source`

### Notify maintainers

@abbradar @tfmoraes 
---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

https://github.com/NixOS/nixpkgs/issues/374878#issuecomment-2758233431

I think I did not provide enough context.
When I made the pr the pr was working on unstable and master branch, but not on 24.11
I glanced over the fact that the issue was reported on 24.11 since the issue related to `wxwidgets`
was not present in newer branches at that time.

The newer issue I was talking about not the same, the issue I found today after your comment is (see the log):
```
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/l470rjcpwgg171bs7wmxfq41z8ard6zn-source
source root is source
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
fixing cmake files...
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/9g3gz3br5q9lmwq55ks5l0wycc2fkk7c-slade-3.2.7/share/loca>
-- The C compiler identification is GNU 14.2.1
-- The CXX compiler identification is GNU 14.2.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- wx-config used is: /nix/store/46rwb13dcgl0z2ql07xbjzp2rmjrply0-wxwidgets-3.2.6/bin/wx-config
-- wxWidgets version is: 3.2.6
-- gtk version is: 3
-- Found wxWidgets: -L/nix/store/46rwb13dcgl0z2ql07xbjzp2rmjrply0-wxwidgets-3.2.6/lib;-pthread;;;-L/nix/store/ylzpk54f82kc888pr4325ynshk48la6i-libglvnd-1.7.0/lib;-L/nix/store/q6063jl4mgbz9fk8hnfy1l5m8y9vldl5-glu-9.0.3/lib;-lwx_gtk3u_aui-3.2;-lwx_gtk3u_gl-3.2;-lwx_gtk3u_stc>
CMake Error at /nix/store/gyn8zrrgzk3h47izzlym76vqz0q2ymvd-sfml-3.0.0/lib/cmake/SFML/SFMLConfig.cmake:182 (message):
  Requested SFML configuration (Shared) was not found
Call Stack (most recent call first):
  cmake/unix.cmake:82 (find_package)
  src/CMakeLists.txt:42 (include)
CMake Error at cmake/unix.cmake:82 (find_package):
  Found package configuration file:
    /nix/store/gyn8zrrgzk3h47izzlym76vqz0q2ymvd-sfml-3.0.0/lib/cmake/SFML/SFMLConfig.cmake
  but it set SFML_FOUND to FALSE so package "SFML" is considered to be NOT
  FOUND.
Call Stack (most recent call first):
  src/CMakeLists.txt:42 (include)
-- Configuring incomplete, errors occurred!
```

--- a/pkgs/games/doom-ports/slade/default.nix
+++ b/pkgs/games/doom-ports/slade/default.nix
@@ -8,7 +8,7 @@
   zip,
   wxGTK,
   gtk3,
-  sfml,
+  sfml_2,
   fluidsynth,
   curl,
   freeimage,
@@ -21,13 +21,13 @@
 
 stdenv.mkDerivation rec {
   pname = "slade";
-  version = "3.2.6";
+  version = "3.2.7";
 
   src = fetchFromGitHub {
     owner = "sirjuddington";
     repo = "SLADE";
     rev = version;
-    hash = "sha256-pcWmv1fnH18X/S8ljfHxaL1PjApo5jyM8W+WYn+/7zI=";
+    hash = "sha256-+i506uzO2q/9k7en6CKs4ui9gjszrMOYwW+V9W5Lvns=";
   };
 
   nativeBuildInputs = [
@@ -41,7 +41,7 @@ stdenv.mkDerivation rec {
   buildInputs = [
     wxGTK
     gtk3
-    sfml
+    sfml_2
     fluidsynth
     curl
     freeimage
@@ -64,11 +64,11 @@ stdenv.mkDerivation rec {
     )
   '';
 
-  meta = with lib; {
+  meta = {
     description = "Doom editor";
     homepage = "http://slade.mancubus.net/";
-    license = licenses.gpl2Only; # https://github.com/sirjuddington/SLADE/issues/1754
-    platforms = platforms.linux;
-    maintainers = with maintainers; [ abbradar ];
+    license = lib.licenses.gpl2Only; # https://github.com/sirjuddington/SLADE/issues/1754
+    platforms = lib.platforms.linux;
+    maintainers = with lib.maintainers; [ abbradar ];
   };
 }

